sale_stock_medical_prescription/models/medical_prescription_order_line.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class MedicalPrescriptionOrderLine(models.Model):
+    _inherit = 'medical.prescription.order.line'
+
+    dispense_uom_id = fields.Many2one(
+        comodel_name='product.uom',
+        string='Dispense UoM',
+        help='Dispense Unit of Measure',
+        required=True,
+        related='medicament_id.uom_id',
```

Comment:
I don't understand the issue you are describing, but this change is not OK & the original method needs to be added back. 

What context uom are you referring to that is causing issues? We don't use that anywhere in medical IIRC.